content/fr/blog/posts/à-l’écoute-des-utilisateurs-amélioration-de-l’outil-de-recherche-de-prestations-liées-à-la-covid-19.md

```diff
@@ -10,12 +10,12 @@ description: >-
   renseignements personnels qui peuvent s’y faufiler.
 author: 'Adrianne Lee, chercheuse'
 date: 2020-07-06T20:39:16.171Z
-image: /img/cds/adrianne-blog-fr.jpg
+image: https://cds-website-assets-prod.s3.ca-central-1.amazonaws.com/adrianne_blog_fr_bf220fb4f8.jpg
 image-alt: >-
   Illustration d’un service en ligne qui reçoit des commentaires écrits sur son
   utilisabilité de la part de 8 personnes différentes.
 translationKey: blog-ffhc19-feedback
-thumb: /img/cds/thumbnails/adrianne-blog-fr.jpg
+thumb: https://cds-website-assets-prod.s3.ca-central-1.amazonaws.com/small_adrianne_blog_fr_bf220fb4f8.jpg
```

Comment:
It's been a while since I worked with AWS, it seems that this is pointing straight at an s3 bucket in a given region. Would something like cloudfront be beneficial here? (It seems like that's what cloudfront is for, but I also maybe failing to remember properly)

Reply:
Yes, I'm tracking that issue here:
https://github.com/cds-snc/cds-website-cms/issues/7

And will likely be implementing a fix for it this afternoon or tomorrow. I will probably have the PR bot change out the s3 url that strapi uses for the cloudfront one. Once I confirm it's working I'll lock down the s3 bucket so that urls can't be accessed directly.

Reply:
Awesome, thanks for the pointer. LGTM.